build-aux/snap/snapcraft.yaml

```diff
@@ -457,6 +470,12 @@ parts:
         touch "${CRAFT_PART_INSTALL}/fips-build-lp"
       fi
 
+      # TODO: restore capabilities after stripping
```

Comment:
Is not it what happens anyway with any form of packaging? We always declare the capabilities and other special permissions somewhere else. So I am not sure we should make it a TODO. It is fine as it is.

Reply:
dropped the TODO, changed comment wording